### ISSUE TITLE
Avoid calls to print(), use logger methods instead

### ISSUE THREAD

**Issue by blake-riley** [ExcitedStates/qfit-3.0#51]
Printing stuff is useful, but has less control than using `logger.info`/`.debug`/`.warning`/`.error`.

I'd like to let the user have slightly more control over how much information we give them.
Low priority, but something to think about while we're working through the existing code.

**Comment by blake-riley** [comment]
Thinking about how this should be done:
- `--verbose` flag will lower logging threshold to `logging.INFO`
- `--debug` flag will lower logging threshold to `logging.DEBUG`

**Comment by stephaniewanko** [comment]
Agree with the --verbose/--debug flags. I think their should be some stuff always written into the INFO file (such as inputs)

**Comment by blake-riley** [comment]
Yup. Definitely making sure the `sys.argv[1:]` gets logged.

**Comment by blake-riley** [comment]
Logging across multiple processes is not trivial.
My preferred method just got blocked by a bug in the Python multiprocessing library (bpo-30256).

**Comment by blake-riley** [comment]
Closed in qfit_protein by #58
